sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -583,7 +626,8 @@ def expand(self, pcoll):
          | "RemoveTempTables/PassTables" >> beam.FlatMap(
              lambda x, deleting_tables: deleting_tables,
              pvalue.AsIter(temp_tables_pc))
-         | "RemoveTempTables/DeduplicateTables" >> Count.PerElement()
+         | "RemoveTempTables/AddUselessValue" >> beam.Map(lambda x: (x, None))
```

Comment:
This looks like the `Distinct` transform:
https://github.com/apache/beam/blob/3b839c8bd6ff849226e325cfc3768169ec061023/sdks/python/apache_beam/transforms/util.py#L198-L203
Although Distinct looks like a heavier operation, so probably best to leave this as is.